Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -1670,7 +1670,8 @@ CanvasUsesAcceleratedDrawing:
       "PLATFORM(IOS_FAMILY) && !PLATFORM(IOS_FAMILY_SIMULATOR)": true
       default: false
     WebKit:
-      default: true
+      "PLATFORM(COCOA) || USE(SKIA)": true
+      default: false
```

Comment:
This setting is used only in `#if USE(CA) || USE(SKIA)`. How about adding `condition: #if USE(CA) || USE(SKIA)`?
https://github.com/WebKit/WebKit/blob/e4d5fc8e99945e2e1e6aed054d54fddb399c1a47/Source/WebCore/html/CanvasBase.cpp#L274-L275

Reply:
OK, good find.

Reply:
Updated.